src/gpu/sycl/ref_prelu.cpp

```diff
@@ -0,0 +1,147 @@
+/*******************************************************************************
+* Copyright 2022 Intel Corporation
```

Comment:
Please update copyrights -> 2023

Reply:
updated copyrights -> 2023 in al the files of PReLU, i.e., ref_prelu.cpp, ref_prelu.hpp, prelu_kernels.hpp and sycl_primitive_conf.hpp